docs/guide/docs/main/emulator.md

```diff
@@ -1,35 +1,53 @@
 ---
 id: emulator
-title: Emulator
+title: Testing Conversations
 ---
 
-You can debug your bot conversation by using the built-in Emulator Window. The emulator is meant for developers to debug the conversations and is not meant to replicate what the end-users will see visually (i.e. some messages will not render properly). To visualize the full user experience, you should use the Web Channel directly.
+You can debug your bot conversation by using the built-in Emulator Window. The emulator is meant for developers to debug the conversations and is not meant to replicate what the end-users see visually (i.e., some messages will not render correctly). To visualize the entire user experience, you should use the Web Channel directly.
```

Comment:
You might want to point out a doc section that shows how to see that "web channel directly"